codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/protocol/ProtocolParserGenerator.kt

```diff
@@ -240,11 +240,15 @@ class ProtocolParserGenerator(
         rust("let mut output = #T::default();", symbolProvider.symbolForBuilder(outputShape))
         if (outputShape.id == operationShape.output.get()) {
             structuredDataParser.operationParser(operationShape)?.also { parser ->
-                rust(
-                    "output = #T(_response_body, output).map_err(#T::unhandled)?;",
-                    parser,
-                    errorSymbol,
-                )
+                // Don't deserialize non-event stream members for an event stream operation with RPC bound protocols,
+                // as they need to be deserialized from payload in the first frame of event stream.
```

Comment:
Would it be possible to link to the relevant specifications here?

Reply:
Added link in 55dda45.